### DISCUSSION TITLE
can rustdesk support remote control on ios device

### DISCUSSION THREAD

**Discussion by xiahc** [rustdesk/rustdesk#10193] [Q&A]
dears
i want use pc to remote control a ios device, if rustdesk can support ,please show me a user guide.
in the download page(https://github.com/rustdesk/rustdesk/releases/tag/1.3.4) ,i only find a iOS TeslFlight, it seems like a controll terminal, not a controlled terminal.
looking forward to reply

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/4839
